### DISCUSSION TITLE
What 64Gb SD cards can be used?

### DISCUSSION THREAD

**Discussion by rrlevy** [tesla-android/issue-tracker#107] [Q&A] [answered]
I've been trying to upgrade my Tesla Android version 2022.44.1 with no luck.

The image won't fit my SD cards... They are SanDisk Ultra 64Gb, but in practice they appear as 63.9 Gb on Raspbery Pi Imager and it says the image requires 64.1 Gb

What brand of cards are you using?

![image](https://user-images.githubusercontent.com/14061585/199137752-57df457d-4fd5-4c97-8c96-3f2bd0c70ef5.png)

**Comment by rrlevy** [comment]
![image](https://user-images.githubusercontent.com/14061585/199137901-b116d1c9-79e8-466f-a6d3-3c43f59a6f99.png)

![image](https://user-images.githubusercontent.com/14061585/199137883-4bdd5e6f-54f0-4751-8eda-ada0e710a364.png)

**Reply by baronbuzby** [reply]
Using a 64gb Samsung EVO

**Comment by bryanbartow** [comment]
@rrlevy Grab the 2022.44.2 hot fix and you should be good to go. Worked great for me.

**Comment by rrlevy** [comment, marked as answer]
Hi, the release was updated and now, as of  2022.44.2, the image fits in the Sandisk!

**Reply by mikegapinski** [reply]
I've made this image smaller in 44.2. I will improve the packaging process for the single image setup down the road, I figured that I could make something like this a day before the build went out and it was a little bit rough ;)

**Comment by inigoml** [comment]
Fixed in 44.2 but same problem in 45.1 withe same Sandisk 64GB card.
I've installed 44.2 meanwhile to test.

**Reply by mikegapinski** [reply]
45.1 image was generated in a exact same way as 44.2, redownload it - it has to work

**Reply by inigoml** [reply]
I'm afraid no... card is OK. In fact I've flashed 44.2 with no issues.
However, with 45.2.... no luck. It crashes at 100%
![image](https://user-images.githubusercontent.com/7486708/206919747-b70aeacb-14e4-4ad9-a366-0ededa0375c0.png)
![image](https://user-images.githubusercontent.com/7486708/206920404-bcbde24c-6a73-4b84-94fb-b9ff6ed3b404.png)
![image](https://user-images.githubusercontent.com/7486708/206920435-eed5d2dd-edc5-4eb8-8896-61555f2cb60c.png)
![image](https://user-images.githubusercontent.com/7486708/206920647-671731a7-90ab-486e-89cf-ebc436b6061b.png)

**Comment by inigoml** [comment]
I will follow this workaround.

https://github.com/tesla-android/issue-tracker/issues/174

**Reply by inigoml** [reply]
Confirmed. Just unzip and flash image.
